src/main/java/mate/academy/Main.java

```diff
@@ -6,18 +6,25 @@
 import mate.academy.service.AuthenticationServiceImpl;
 import mate.academy.service.OrderService;
 import mate.academy.service.OrderServiceImpl;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class Main {
+    private static final Logger logger = LogManager.getLogger(Main.class);
+
     public static void main(String[] args) {
         AuthenticationService authenticationService = new AuthenticationServiceImpl();
         User user;
+        String login = "bob";
         try {
             user = authenticationService.login("bob", "1234");
+            logger.info("User logged in successfully {}", user.getLogin());
         } catch (AuthenticationException e) {
-            e.printStackTrace();
+            logger.error("Authentication failed for user: bob", e);
```

Comment:
The log message should include the login parameter to make it more informative as per the checklist. For example: logger.error("Authentication failed for user: {}", login, e);